lib/sequenceserver/blast/report.rb

```diff
@@ -87,7 +87,8 @@ def extract_program_info(ir)
       def extract_params(ir)
         params = Hash[
           *ir[7].first.map { |k, v| [k.gsub('Parameters_', ''), v] }.flatten
-        ].merge(job.advanced_params)
+        ]
+        params = params.merge(job.advanced_params) unless job.advanced_params.nil?
```

Comment:
I am curious if this change is precautionary or if nil is actually returned. If so, I think we should instead make sure job.advanced_params returns empty Hash instead of nil.